core/common/src/main/java/alluxio/uri/StandardURI.java

```diff
@@ -218,31 +199,10 @@ public boolean equals(Object o) {
     if (this.mPath == null) {
       return URIUtils.equals(this.mSchemeSpecificPart, that.mSchemeSpecificPart);
     }
-    if (!URIUtils.equals(this.mPath, that.mPath)) {
-      return false;
-    }
-    if (!URIUtils.equals(this.mQuery, that.mQuery)) {
-      return false;
-    }
 
-    if (this.mAuthority == that.mAuthority) {
-      return true;
-    }
-    if (this.mHost != null) {
-      // host-based authority
-      if (that.mHost == null) {
-        return false;
-      }
-      if (this.mHost.compareToIgnoreCase(that.mHost) != 0) {
-        return false;
-      }
-      if (this.mPort != that.mPort) {
-        return false;
-      }
-    } else if (!URIUtils.equals(this.mAuthority, that.mAuthority)) {
-      return false;
-    }
-    return true;
+    return URIUtils.equals(this.mPath, that.mPath)
+        && URIUtils.equals(this.mQuery, that.mQuery)
+        && URIUtils.equals(this.mAuthority.toString(), that.mAuthority.toString());
```

Comment:
use the equals implemented on `Authority`

Reply:
Changed to 
```
    return URIUtils.equals(this.mPath, that.mPath)
        && URIUtils.equals(this.mQuery, that.mQuery)
        && this.mAuthority.equals(that.mAuthority);
```
thanks